services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/nodeports.py

```diff
@@ -243,14 +248,32 @@ def _is_zip_file(file_path: Path) -> bool:
 _shutil_move = aiofiles.os.wrap(shutil.move)
 
 
+async def _move_file_to_input_port(
+    final_path: Path, downloaded_file: Path, dest_folder: PrunableFolder
+) -> None:
+    _logger.debug("moving %s", downloaded_file)
```

Comment:
please use ```with log_context``` instead of these debug messages. and it gives you a logger inside the context as well